tls/s2n_post_handshake.c

```diff
@@ -79,7 +79,9 @@ int s2n_post_handshake_send(struct s2n_connection *conn, s2n_blocked_status *blo
     POSIX_ENSURE_REF(conn);
 
     POSIX_GUARD(s2n_key_update_send(conn, blocked));
-    POSIX_GUARD_RESULT(s2n_tls13_server_nst_send(conn, blocked));
+    if(s2n_result_is_error(s2n_tls13_server_nst_send(conn, blocked))) {
```

Comment:
I think this function also tries to flush the ticket on the wire, right? Will ignoring things like `EAGAIN` here be handled properly later? Are there any other critical errors where we might want to fail (like error in the middle of `s2n_record_writev`?

nit: I think the rest of the code base uses space between `if` and `(`
```suggestion
    if (s2n_result_is_error(s2n_tls13_server_nst_send(conn, blocked))) {
```

Reply:
Good point, I'll move the error handling.